### PR TITLE
[SDK] Add some missing headers / definitions

### PR DESCRIPTION
**Pull request by tkreuzer**
## Purpose

Add some missing headers / definitions.

### CODE REVIEW

File: sdk/include/psdk/appmodel.h [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+typedef enum AppPolicyWindowingModel
+{
+    AppPolicyWindowingModel_None = 0,
+    AppPolicyWindowingModel_Universal = 1,
+    AppPolicyWindowingModel_ClassicDesktop = 2,
+    AppPolicyWindowingModel_ClassicPhone = 3
+} AppPolicyWindowingModel;
+
+#ifdef __cplusplus
+extern "C" {
```
Comment (HBelusca) [inline sdk/include/psdk/appmodel.h:31]: Shouldn't the typedef'ed enums be inside the extern C as well?

File: sdk/include/psdk/appmodel.h [hunk 2]
```diff
@@ -0,0 +1,72 @@
+
```
Comment (HBelusca) [inline sdk/include/psdk/appmodel.h:1]: These files are missing a license header.

File: sdk/include/psdk/stringapiset.h [hunk 3]
```diff
@@ -0,0 +1,19 @@
+
+#pragma once
+
+#include <winnls.h>
+
+WINBASEAPI
```
Comment (HBelusca) [inline sdk/include/psdk/stringapiset.h:6]: No extern C ?

File: sdk/include/psdk/appmodel.h (hunk 1 above)
Comment (tkreuzer) [reply sdk/include/psdk/appmodel.h:31]: POD types do not have a "linkage", so it makes no difference. It is different for classes with member functions and template types, which need to be `extern "C++"`.

File: sdk/include/psdk/appmodel.h (hunk 2 above)
Comment (tkreuzer) [reply sdk/include/psdk/appmodel.h:7]: Added

File: sdk/include/psdk/stringapiset.h (hunk 3 above)
Comment (tkreuzer) [reply sdk/include/psdk/stringapiset.h:16]: Fixed

**Review by HBelusca — APPROVED** [review]

**Review by whindsaks — APPROVED** [review]

**Review by GeoB99 — APPROVED** [review]

**Review by DarkFire01 — APPROVED** [review]

**Pull request merged**